app/src/main/java/ch/sdp/vibester/activity/BuzzerScreenActivity.kt

```diff
@@ -43,28 +49,137 @@ class BuzzerScreenActivity : AppCompatActivity() {
         supportActionBar?.hide()
         setContentView(R.layout.activity_buzzer_screen)
 
+        val ctx: Context = this
+
         val getIntent = intent.extras
-        val nPlayers = getIntent?.getInt("Number of players")
+        if (getIntent != null) {

```

Comment:
rename getIntent to intentExtras or something like that

Reply:
I am not sure whether it is okay to keep all code under if statement. In other activities we just check if the extra exists.